hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -113,11 +115,14 @@ public void initialize(URI name, Configuration conf) throws IOException {
     String omPort = String.valueOf(-1);
     if (!isEmpty(remaining)) {
       String[] parts = remaining.split(":");
-      if (parts.length != 2) {
+      // Array length should only be 1 or 2
+      if (parts.length > 2) {
         throw new IllegalArgumentException(URI_EXCEPTION_TEXT);
```

Comment:
We need to update the URI_EXCEPTION_TEXT to add about the new support of o3fs://bucket.volume:om-host.

Reply:
On it.

Reply:
Done. Updated `URI_EXCEPTION_TEXT`.